src/documents/SettingsStore.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Settings Store
+layout: default
+category: Infusion
+---
+
+The Infusion [Preferences Framework](PreferencesFramework.md) uses a _Settings Store_ to read and
+write preferences.
+
+## fluid.prefs.store ##
+
+A base grade to be used by any preference setting store.
+This component uses the following base [grades](ComponentGrades.md):
+
+* [`fluid.prefs.dataSource`](#fluid-prefs-datasource)
+* [`fluid.contextAware`](ContextAwareness.md)
+
+
+## fluid.prefs.dataSource ##
+
+A grade used by `fluid.prefs.store`; defines placeholders for `get` and `set` methods.
+
+Method Placeholder  | Description
+------------------- | -----------
+`get`  | For retrieving preferences. Should return the preferences
+`set`  | For saving preferences. Should accept the preferences as an argument.
+
+Note that no particular parameters are prescribed for these methods. It is up to the developer
```

Comment:
I've gone through and found some other notes to style this way, though there were a few that I thought didn't warrant the treatment.